huey_monitor/admin.py

```diff
@@ -27,11 +43,7 @@ def get_changelist(self, request, **kwargs):
         return TaskModelChangeList
 
     def column_name(self, obj):
-        qs = TaskModel.objects.filter(parent_task_id=obj.pk).order_by('-create_dt')
-        context = {
-            'main_task': obj,
-            'sub_tasks': qs
-        }
+        context = {'main_task': obj, 'sub_tasks': obj.sub_tasks.all()}
```

Comment:
@henribru, if you don't mind, I would ask you to keep `qs` separate and define `'sub_tasks': qs` within `context` because of upcoming modification I will soon propose as part of another PR.
ideally:
```
        qs = obj.sub_tasks.all()
        context = {
            'main_task': obj,
            'sub_tasks': qs
        }
```